docs/articles/retrieval_augmented_generation_eval_qdrant_arize.md

```diff
@@ -0,0 +1,519 @@
+
+```python
+## Indexing the vectors into Qdrant collection 
+from phoenix.trace import suppress_tracing
+with suppress_tracing():
+  dense_vector_index = VectorStoreIndex.from_documents(
+      documents,
+      storage_context=storage_context,
+      show_progress=True
+  )
+```
+
+Once the data is pushed in, we will create our `dense-vector retriever` as below :
+
+```python
+##Initialise retriever to interact with the Qdrant collection
+dense_retriever = VectorIndexRetriever(
+    index=dense_vector_index,
+    vector_store_query_mode=VectorStoreQueryMode.DEFAULT,
+    similarity_top_k=2
+)
+```
+
+which we’ll use through Phoenix to evaluate our Naive Rag pipeline.
+
+We'll use the pre-compiled **[hugging-face dataset](https://huggingface.co/datasets/atitaarora/qdrant_doc)**,which consists of `text` and `source`, derived from the documentation as before in our previous article, for the evaluation.
+
+Let’s load our eval dataset as :
+
+```python
+## Loading the Eval dataset
+from datasets import load_dataset
+qdrant_qa = load_dataset("atitaarora/qdrant_doc_qna", split="train")
+qdrant_qa_question = qdrant_qa.select_columns(['question'])
+qdrant_qa_question['question'][:10]
+
+#Outputs
+['What is vaccum optimizer ?',
+ 'Tell me about ‘always_ram’ parameter?',
+ 'What is difference between scalar and product quantization?',
+ 'What is ‘best_score’ strategy?',
+ 'How does oversampling helps?',
+ 'What is the purpose of ‘CreatePayloadIndexAsync’?',
+ 'What is the purpose of ef_construct in HNSW ?',
+ 'How do you use ‘ordering’ parameter?',
+ 'What is significance of ‘on_disk_payload’ setting?',
+ 'What is the impact of ‘write_consistency_factor’ ?']
+
+```
+
+Next , we will define our response synthesizer and associate it with  `query engine` which will facilitate collection of traces for the evaluations.
+
+```python
+from llama_index.core import get_response_synthesizer
+from llama_index.core.query_engine import RetrieverQueryEngine
+
+#define response synthesizer
+response_synthesizer = get_response_synthesizer()
+
+#assemble query engine for dense retriever
+dense_query_engine = RetrieverQueryEngine(
+                     retriever=dense_retriever,
+                     response_synthesizer=response_synthesizer,)
+#query_engine = index.as_query_engine()
+for query in tqdm(qdrant_qa_question['question'][:10]):
+    try:
+      dense_query_engine.query(query)
+    except Exception as e:
+      pass
+```
+
+Open the Phoenix UI with the [link](http://localhost:6006/) and click through the queries to better understand how the query engine is performing.
+
+Check the Phoenix UI as your queries run. Your traces should appear in real time.
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/naive_rag_run1.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/naive_rag_run1.png)
+
+For each trace notice a span breakdown and the list of all the spans can also be seen by clicking on spans tab as below :
+
```

Comment:
I see span_kind = LLM. Where do the spans start and stop?

Reply:
This is internal to tool (Arize Phoenix) not a user controlled parameter.

Reply:
@atarora
Your text says "for each trace, notice a span breakdown"
Before going to the spans tab, how do we "notice a span breakdown" for each trace? Try to be very specific about how we notice this.